docs/components/tools.md

```diff
@@ -813,7 +813,7 @@ Flags:
       --rewrite.to-relabel-config-file=<file-path>
                                 Path to YAML file that contains relabel configs
                                 that will be applied to blocks
-      --tmp.dir="/tmp/thanos-rewrite"
+      --tmp.dir="/var/folders/7w/bk4g23r116j_srlrlf8_ys7r0000gp/T/thanos-rewrite"
```

Comment:
Mind reverting this? I think our doc generate tool has an issue on MacOS.

Reply:
yep, reverted